src/Microsoft.Azure.EventHubs.Processor/EventHubPartitionPump.cs

```diff
@@ -33,21 +33,9 @@ public EventHubPartitionPump(EventProcessorHost host, Lease lease)
                 catch (Exception e)
                 {
                     lastException = e;
-                    if (e is ReceiverDisconnectedException)
-                    {
-                        // TODO Assuming this is due to a receiver with a higher epoch.
-                        // Is there a way to be sure without checking the exception text?
-                        ProcessorEventSource.Log.PartitionPumpWarning(
-                            this.Host.Id, this.PartitionContext.PartitionId, "Receiver disconnected on create, bad epoch?", e.ToString());
-                        // If it's a bad epoch, then retrying isn't going to help.
-                        break;
-                    }
```

Comment:
Removing this part since CreateReceiver doesn't throw ReceiverDisconnectedException since link is created with the first Receive call not here.